Lib/fontParts/base/bPoint.py

```diff
@@ -22,15 +22,15 @@ def _setPoint(self, point):
     def __eq__(self, other):
         if hasattr(other, "_point"):
             return self._point == other._point
-        return False
+        return NotImplemented
```

Comment:
__eq__ shall only return True or False.

Reply:
"A rich comparison method may return the singleton NotImplemented if it does not implement the operation for a given pair of arguments." from the python documentation, so it falls back to NotImplemented when it can't compare two objects that are of unlike types, but I can change that back.

Reply:
I thought that only applies to __lt__ etc. But you are right. Python takes care of it.

Reply:
I made that switch when I put in `__ne__` for 2.7, as returning `True` if it can't compare objects didn't make sense.